vertical-pod-autoscaler/pkg/recommender/model/aggregations_config.go

```diff
@@ -83,3 +94,10 @@ func memoryHistogramOptions() util.HistogramOptions {
 	}
 	return options
 }
+
+func init() {
+	flag.DurationVar(&MemoryAggregationWindowLength, "memory-aggregation-window-length", DefaultMemoryAggregationWindowLength, `The length of the memory usage history aggregated by VPA.`)
+	flag.DurationVar(&MemoryAggregationInterval, "memory-aggregation-interval", DefaultMemoryAggregationInterval, `The length of a single interval, for which the peak memory usage is computed. Memory usage peaks are aggregated in daily intervals. In other words there is one memory usage sample per day (the maximum usage over that day). Note: AggregationWindowLength must be integrally divisible by this value.`)
```

Comment:
"Memory usage peaks are aggregated in daily intervals." is not true any more.

Reply:
Copy-paste problem (copied from the variable comments) 😅 . I will change it.

Reply:
[Fixed](https://github.com/kubernetes/autoscaler/pull/2905/files#diff-483c01ba6f34fd6f8c9d4944684cff98R54).